CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Master (Unreleased)
 
```

Comment:
Needs rebasing after 3.0 release?

Reply:
This is rubocop-factory_bot, not rubocop-rspec :-)